modules/end-user-guide/partials/assembly_building-images-for-ides-based-on-the-intellij-platform.adoc

```diff
@@ -0,0 +1,18 @@
+:parent-context-of-building-images-for-ides-based-on-the-intellij-platform: {context}
+
+[id="building-images-for-ides-based-on-the-intellij-platform_{context}"]
+= Building images for IDEs Based on the IntelliJ Platform
```

Comment:
```suggestion
= Building images for IntelliJ Platform-based IDEs
```

Reply:
Let's keep the same formulation everywhere: IDEs Based on the IntelliJ Platform.

Reply:
No capitalization:

_IDEs based on the IntelliJ Platform_